l10n_nl_tax_statement/models/l10n_nl_vat_statement.py

```diff
@@ -72,10 +72,8 @@ def default_get(self, fields_list):
     @api.onchange('date_range_id')
     def onchange_date_range_id(self):
         if self.date_range_id and self.state == 'draft':
-            self.write({
-                'from_date': self.date_range_id.date_start,
-                'to_date': self.date_range_id.date_end,
-            })
+            self.from_date = self.date_range_id.date_start
+            self.to_date = self.date_range_id.date_end
```

Comment:
I didn't now, thank you!

You just found another issue: on this module I created a method called `update()` that does something else, but actually it overrides the standard `update()`. I'm fixing it.